content/news/this-week-in-fluvio-0018.md

```diff
@@ -61,9 +61,47 @@ For more detail on the available config options, see the [Rust docs](https://doc
 
 #### Support for 3rd-Party Connectors
 
-We've improved the experience for running connectors. Previously, the only options for connectors were to run our officially built connectors, or to locally build your own. In this new release, users can set up their connector pipelines and allow for others to use unofficial Fluvio connectors in their own Fluvio clusters.
+We've improved the experience for running connectors. Previously, the connectors available to run were to run our officially built connectors, or you could [build your own]( {{<ref "/connectors/developer-guide/overview">}}). In this new release, users who have built their own connectors and published the image(s) to Docker Hub can share those **pre-built** connector images with others.
 
-More information about this feature will be documented soon! Until then, please reach out on our Discord for assistance getting started.
+We'll walk through an example scenario [from our tests](https://github.com/infinyon/fluvio/blob/master/tests/cli/smoke_tests/connector-3rd-party.bats).
+
+First You need to start your cluster with the **HTTP URL prefix** where the YAML config for your the 3rd party image source is located.
+
+We'll be using the URL "[https://raw.githubusercontent.com/infinyon/fluvio-connectors](https://raw.githubusercontent.com/infinyon/fluvio-connectors)" (Github's HTTP source to raw content of our repo, [fluvio-connectors](https://github.com/infinyon/fluvio-connectors)).
+
+%copy%
+```bash
+$ fluvio cluster start --connector-prefix "https://raw.githubusercontent.com/infinyon/fluvio-connectors"
+```
+
+For our connectors to use an arbitrary image location, we specify a URL to a YAML config in the `type` field of in our connector config `3rd-party-test-connector-config.yaml`. Notice that the start of the URL matches the `--connector-prefix` we defined when we started the cluster.
+
+%copy%
+```yaml
+# 3rd-party-test-connector-config.yaml
+version: v1
+name: my-third-party-connector
+type: "https://raw.githubusercontent.com/infinyon/fluvio-connectors/main/rust-connectors/utils/test-connector/connector.yaml"
+topic: my-test-connector-topic
+create_topic: true
+direction: source
+```
+
+The YAML config at the URL only has one field `image`. The value is the docker hub image name.
```

Comment:
I don't see the `image` field in the `yaml` above.

Reply:
It was in the code block below, but I've cleared this up